models/comments.models.js

```diff
@@ -1,2 +1,13 @@
 const db = require('../db/connection')
 
+exports.selectCommentsByArticleId = (article_id) => {
+    return db.query("SELECT * FROM comments JOIN articles ON articles.article_id = comments.article_id WHERE comments.article_id = $1;", [article_id])
+    .then((results) => {
+        if(!results.rows[0]){
+            return Promise.reject({status: 404, msg: "Not found"})
```

Comment:
what will happen what we invoke this model with article_id for an article that has no comments?  We need to check that the article we want comments for exists, and only send an error if it doesn't.  The lecture today covered this if you were around!